APMrover2/failsafe.cpp

```diff
@@ -5,6 +5,8 @@
 
 #include "Rover.h"
 
+#include <stdio.h>
```

Comment:
I see the other vehicles all have this include as well although I'm surprised we need it.